alpaca/common/models/trading.py

```diff
@@ -1,28 +1,184 @@
+        replaced_by (Optional[UUID]): ID of Order that replaces this Order.
+        replaces (Optional[UUID]): ID of Order which this Order replaces.
+        asset_id (UUID): ID of the asset.
+        symbol (str): Symbol of the asset.
+        asset_class (str): Asset class of the asset.
+        notional (Optional[str]): Ordered notional amount. If entered, qty will be null. Can take up to 9 decimal
+          points.
+        qty (Optional[str]): Ordered quantity. If entered, notional will be null. Can take up to 9 decimal points.
+        filled_qty (Optional[str]): Filled quantity.
+        filled_avg_price (Optional[str]): Filled average price. Can be 0 until order is processed in case order is
+          passed outside of market hours.
+        order_class (OrderClass): Valid values: simple, bracket, oco or oto.
+        order_type (OrderType): Deprecated with just type field below.
+        type (OrderType): Valid values: market, limit, stop, stop_limit, trailing_stop.
+        side (OrderSide): Valid values: buy and sell.
+        time_in_force (TimeInForce): Length of time the Order is in force.
+        limit_price (Optional[str]): Limit price of the Order.
+        stop_price (Optional[str]): Stop price of the Order.
+        status (OrderStatus): The status of the Order.
+        extended_hours (bool): If true, eligible for execution outside regular trading hours.
+        legs (Optional[List[Order]]): When querying non-simple order_class orders in a nested style, an array of Order
+          entities associated with this order. Otherwise, null.
+        trail_percent (Optional[str]): The percent value away from the high water mark for trailing stop orders.
+        trail_price (Optional[str]): The dollar value away from the high water mark for trailing stop orders.
+        hwm (Optional[str]): The highest (lowest) market price seen since the trailing stop order was submitted.
+        commission (float): The dollar value commission you want to charge the end user.
+    """
+
+    id: UUID
+    client_order_id: UUID
+    created_at: datetime
+    updated_at: datetime
+    submitted_at: datetime
+    filled_at: Optional[datetime]
+    expired_at: Optional[datetime]
+    canceled_at: Optional[datetime]
+    failed_at: Optional[datetime]
+    replaced_at: Optional[datetime]
+    replaced_by: Optional[UUID]
+    replaces: Optional[UUID]
+    asset_id: UUID
+    symbol: str
+    asset_class: str
+    notional: Optional[str]
+    qty: Optional[str]
+    filled_qty: Optional[str]
+    filled_avg_price: Optional[str]
+    order_class: OrderClass
+    order_type: OrderType
+    type: OrderType
+    side: OrderSide
+    time_in_force: TimeInForce
+    limit_price: Optional[str]
+    stop_price: Optional[str]
+    status: OrderStatus
+    extended_hours: bool
+    legs: Optional[List["Order"]]
+    trail_percent: Optional[str]
+    trail_price: Optional[str]
+    hwm: Optional[str]
+    commission: float

```

Comment:
Let's leave out commission. Broker will have its own Order object extended from this one that can implement commission and any other broker related fields.